recipe/meta.yaml

```diff
@@ -75,44 +78,59 @@ outputs:
       host:
         - llvmdev {{ version }}
         - llvm {{ version }}
-        - zlib
```

Comment:
We don't need zlib?

Reply:
apparently not. Build succeeds without. I'm not sure why conda-forge has it in